packages/graphql-mocks/src/graphql/handler.ts

```diff
@@ -16,10 +17,14 @@ export class GraphQLHandler {
   protected graphqlSchema: GraphQLSchema;
   protected initialContext: GraphQLArgs['contextValue'];
   protected initialResolverMap: ResolverMap;
+  protected resolverMap?: ResolverMap;
   protected scalarMap: ScalarMap;
 
   constructor(options: CreateGraphQLHandlerOptions) {
-    const graphqlSchema = createSchema(options.dependencies?.graphqlSchema);
+    const { modifySchemaDepenency } = options;
+    const graphqlSchema = createSchema(options.dependencies?.graphqlSchema, {
+      makeCopy: !modifySchemaDepenency,
```

Comment:
copy vs no copy appears to help

Reply:
Coming back to this, it probably isn't worth doing now. A huge schema still copies fairly quickly. Can always revisit later if this ends up making a difference.